examples/platform/silabs/BaseApplication.cpp

```diff
@@ -45,6 +45,15 @@
 #endif // CHIP_CONFIG_ENABLE_ICD_SERVER
 
 #include <app/server/OnboardingCodesUtil.h>
+
+#if SL_BLE_SIDE_CHANNEL_ENABLED
```

Comment:
Pre-processor macro are not uniformed in this file, here it's #if and at line 137 you also check if the define is actually defined. Please unify the check across the file